db.class.php

```diff
@@ -597,6 +618,15 @@ public function queryRawUnbuf() { $args = func_get_args(); return $this->prepend
   protected function queryHelper() {
     $args = func_get_args();
     $type = array_shift($args);
+
+    $isMultiUpdate = false;
+
+    if (!empty($args) && $args[1] === 'multiUpdate') {
```

Comment:
@DrParanoia Found a bug. This is causing notices when the count($args) === 1. Undefined index.

Reply:
@artursvonda is it critical? 😆 Or can we leave it when we next time "patch" MeekroDB?